main/src/ca/uwaterloo/flix/language/phase/unification/TraitEnv.scala

```diff
@@ -16,7 +16,7 @@
 package ca.uwaterloo.flix.language.phase.unification
```

Comment:
What is the difference between TraitEnv and TraitEnvironment? Any chance at better names?

I suppose at least one is in the AST and one is in the solver.

Reply:
TraitEnvironment will die with the old solver.